### ISSUE TITLE
Support Drizzle ORM instead of Prisma

### ISSUE THREAD

**Issue by louisjrdev** [sidebase/sidebase#112]
### Describe the feature

Would you consider adding support for Drizzle ORM instead of Prisma? 

It has gained some great traction in the community and would be great to see it in sidebase.

**Comment by phoenix-ru** [comment]
Yes, this is a planned feature. We are internally outlining the customizable `create-sidebase` template. Can't provide estimates though, as this is a bit lower on our list.
Your PRs are welcome!
